### ISSUE TITLE
NotFoundHttpException's message in crud should use i18n

### ISSUE THREAD

**Issue by nkovacs** [yiisoft/yii2-gii#174]
### What steps will reproduce the problem?
1. Open the CRUD Generator
2. Check "Enable I18N"
3. Fill out the rest of the form
4. Click preview
### What's expected?

The generated controller's `findModel` method throws a NotFoundHttpException with the message 'The requested page does not exist.'. This should be translated, since it'll be displayed to the user on the error page.
### What do you get instead?

An untranslatable string:

```
throw new NotFoundHttpException('The requested page does not exist.');
```
### Additional info

The code in question:
https://github.com/yiisoft/yii2-gii/blob/master/generators/crud/default/controller.php#L173

| Q | A |
| --- | --- |
| Yii vesion | 2.0.8 |
| PHP version | 5.6.22 |
| Operating system | Debian 8 |

**Comment by samdark** [comment]
Agree.

**Comment by cebe** [comment]
fixed by #306
